projects/sandbox/src/app/collapsible/collapsible.component.html

```diff
@@ -1,6 +1,6 @@
 <tree-node>
    <div class="container">
-      <div class="collapse-button" (click)="toggle()">
+      <button class="collapse-button" (click)="toggle()">
```

Comment:
```suggestion
      <div role="button" class="collapse-button" (click)="toggle()">
```

question: could you, maybe, do this instead and remove the need for the CSS updates?

Reply:
Never mind. That probably wouldn't be good cause the element would still not be focusable or respond correctly to key presses.

Reply:
yeah I belieev `button` is semantically correct. I see some places in webapp where the other teams are using 

`<div role="button" tabIndex="0" (click)... (keyup.Enter)... (keyup.Space)... >`

when they could be using `button`